2018 Robot/src/org/usfirst/frc/team4999/utils/MoPrefs.java

```diff
@@ -0,0 +1,64 @@
+package org.usfirst.frc.team4999.utils;
+
+import edu.wpi.first.wpilibj.Preferences;
+
+public class MoPrefs {
+	
+	private static Preferences prefs = Preferences.getInstance();
+	
+	private static final double AUTO_SPEED = 0.25; 				// Maximum speed the robot will travel at in autonomous
+	private static final double DRIVE_ENC_TICKS = 562.666378; 	// Number of encoder ticks per meter of travel
```

Comment:
Is this calculated from theoretical values or empirical measurements?  I highly doubt the number of sig figs here.